[quant][docs] Move some of the descriptions out of codeblock

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #89804
* __->__ #89795

Summary:
This is to make sure the description texts are wrapping around code, instead of being displayed as a single line

Test Plan:
visual inspections

Reviewers:

Subscribers:

Tasks:

Tags:

cc @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

jerryzh168 committed:
[quant][docs] Move some of the descriptions out of codeblock

Summary:
This is to make sure the description texts are wrapping around code, instead of being displayed as a single line

Test Plan:
visual inspections

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

andrewor14 commented:
<img width="1043" alt="Screen Shot 2022-11-28 at 7 09 20 PM" src="https://user-images.githubusercontent.com/2133137/204406789-3ee61b57-6fc7-4630-b151-1366c930cc84.png">

Looks like some sentences still need their own paragraphs

jerryzh168 committed:
Update on "[quant][docs] Move some of the descriptions out of codeblock"


Summary:
This is to make sure the description texts are wrapping around code, instead of being displayed as a single line

Test Plan:
visual inspections

Reviewers:

Subscribers:

Tasks:

Tags:

cc jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen leslie-fang-intel

[ghstack-poisoned]

andrewor14 commented:
nit: sorry, I found two more of these:

<img width="1032" alt="Screen Shot 2022-11-28 at 7 49 42 PM" src="https://user-images.githubusercontent.com/2133137/204411246-c8841d27-1407-481d-b6fb-b16ac6ae29e8.png">

<img width="1038" alt="Screen Shot 2022-11-28 at 7 49 15 PM" src="https://user-images.githubusercontent.com/2133137/204411259-bff08f3c-2e9a-41a8-bd9f-4832dcc4bd2c.png">

jerryzh168 committed:
Update on "[quant][docs] Move some of the descriptions out of codeblock"


Summary:
This is to make sure the description texts are wrapping around code, instead of being displayed as a single line

Test Plan:
visual inspections

Reviewers:

Subscribers:

Tasks:

Tags:

cc jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen leslie-fang-intel

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][docs] Move some of the descriptions out of codeblock"


Summary:
This is to make sure the description texts are wrapping around code, instead of being displayed as a single line

Test Plan:
visual inspections

Reviewers:

Subscribers:

Tasks:

Tags:

cc jianyuh raghuramank100 jamesr66a vkuzo jgong5 Xia-Weiwen leslie-fang-intel

[ghstack-poisoned]
